### ISSUE TITLE
Smtp plugin

### ISSUE THREAD

**Issue by Rupinderwadali** [Rupinderwadali/store#9]
I used a smtp plugin to send an e-mail to user who sign-up to wordpress to provide him password. But it 's not working for me :(. I faced a new error when activate this plugin. The error is:

Plugin could not be activated because it triggered a fatal error.
Please enable 'php_mycrypt' in PHP. It is needed to encrypt passwords.
Fatal error: Could not activate SMTP. in /var/www/html/wordpress/wp-content/plugins/smtp/smtp.php on line 41

To solve this error I install the php mcrypt extension and follow the following tutorial http://aryo.lecture.ub.ac.id/easy-install-php-mcrypt-extension-on-ubuntu-linux/

But still face the same problem and after that I go with a new plugin i.e Postman smtp but after configuring, it again did not solve my problem of sending a e- mail to user for their password.

**Comment by shaina7837** [comment]
Try this
`sudo php5enmod mcrypt`
and then
`sudo service apache2 restart`

**Comment by Rupinderwadali** [comment]
Yeah! now the plugin is activated. But still the problem is not solved, mail is not send to user and I think there is some problem in server. My mentor and me trying to find out the problem.
